### DISCUSSION TITLE
Light bleed compensation (Pixel lookup mode:both)

### DISCUSSION THREAD

**Discussion by nassry** [sn4k3/UVtools#693] [Q&A] [answered]
So I have been playing around with this feature and it has worked wonderful! I've tried the ,,previous'' and also the ,,next'' option, but for ,,both'' I expected it to incoporate the feature of the two previous options. Instead, it dimmed all the pixels to the lowest setting.

**Comment by sn4k3** [comment, marked as answer]
Because it will look bottom and top layers and sum the result to dim the pixels. Use the option that work best for you.
